code/modules/keybindings/keybind/communication.dm

```diff
@@ -30,6 +30,7 @@
 /datum/keybinding/client/communication/say_with_indicator/down(client/user)
 	var/mob/M = user.mob
 	M.say_typing_indicator()
+	M.play_sound_typing_indicator()
```

Comment:
Just have `say_typing_indicator()` run this proc, would make it so you dont need a global var!

Reply:
there's an issue with this, using a single proc makes it impossible to begin saying a new sentence until all sounds have been played